README.md

```diff
@@ -88,6 +88,7 @@ The following 44 devices are supported.
 | [Resistive Touchscreen (4-wire)](http://ww1.microchip.com/downloads/en/Appnotes/doc8091.pdf) | GPIO |
 | [Semihosting](https://wiki.segger.com/Semihosting) | Debug |
 | [Shift register](https://en.wikipedia.org/wiki/Shift_register#Parallel-in_serial-out_\(PISO\)) | GPIO |
```

Comment:
I would rewrite both lines to 
- Shift register (PISO)
- Shift register (SIPO)

So the difference is clear between both options

Reply:
Should I change the other entry in README.md?
I think we  should keep the existing driver's folder name as it is to not break existing code.